packages/api/.gitignore

```diff
@@ -0,0 +1 @@
+prisma/client
```

Comment:
Because the prisma client is generated code, we are preventing it from being committed to version control